frontend/packages/knative-plugin/integration-tests/features/admin-perspective/serverless-admin-empty-state.feature

```diff
@@ -5,6 +5,8 @@ Feature: Serving and Eventing pages at Administrator perspective
 
         Background:
             Given user is at administrator perspective
+              And user has installed OpenShift Serverless Operator
+              And user has deployed Knative Serving component
```

Comment:
Actually we have included the code for knative installation in before hook, because all the scenarios present in Knative plugin this installation is mandatory

For End user understanding purpose, If you are adding these, Lets keep it, but we can comment it out